src/svg-pan-zoom.js

```diff
@@ -30,6 +30,10 @@ var optionsDefaults = {
 , customEventsHandler: null
 , eventsListenerElement: null
 , onUpdatedCTM: null
+, triggerUpOnLeave: true // This allows content to be panned even if the panning pointer is on top of a menu or outside browser
+, restrictPanButton: null // This restricts pan to specific evt.button
```

Comment:
`eventButtonUsedForPanning` // Restricts pan to specific mouse button. Full list https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button